ibm/service/cdtoolchain/resource_ibm_cd_toolchain.go

```diff
@@ -91,7 +91,7 @@ func ResourceIBMCdToolchain() *schema.Resource {
 }
 
 func ResourceIBMCdToolchainValidator() *validate.ResourceValidator {
-	validateSchema := make([]validate.ValidateSchema, 0)
+	validateSchema := make([]validate.ValidateSchema, 1)
```

Comment:
This is the generator issue actually the length should be zero...we raised issue with generator team..can you revert these changes